src/app/pages/ArticlePage/ElectionBanner/index.tsx

```diff
@@ -85,6 +88,8 @@ export default function ElectionBanner({ aboutTags, taggings }: Props) {
     );
   }
 
+  if (variation === 'off') return null;
```

Comment:
The `off` variation is for the `0.5%` (or there abouts?) of people who won't get the banner?

Reply:
Minor but I'd maybe move this up directly below the line 37 so that it immediately bails out if its `off`.

Reply:
Should it go below line 60? As we want the toggle to have precedence?

Reply:
Ah yes you're right. I should have probably separated out the toggle flag as a separate `if` because it should have precedence as you say. But yea, after line 60 is fine.

Reply:
Just to check as well for the 99.5%, is `variation` just undefined/null?

Reply:
Yeah as the experiment wouldn't come through as a header, so useOptimizelyMVtVariation will return null